pkg/watch/watch.go

```diff
@@ -484,7 +496,11 @@ func (o *WatchClient) CleanupDevResources(devfileObj parser.DevfileObj, componen
 	fmt.Fprintln(out, "Cleaning resources, please wait")
 	isInnerLoopDeployed, resources, err := o.deleteClient.ListResourcesToDeleteFromDevfile(devfileObj, "app", componentName, labels.ComponentDevMode)
 	if err != nil {
-		fmt.Fprintf(out, "failed to delete inner loop resources: %v", err)
+		if kerrors.IsUnauthorized(err) {
+			fmt.Fprintf(out, "Error connecting to the cluster, the resources were not cleaned up.\nPlease log in again and cleanup the resource with `odo delete component`\n\n")
+		} else {
```

Comment:
Wondering if we shouldn't display a similar message when we get a Forbidden error.

When trying to reproduce the issue (`oc logout` while a Dev session is running, then stop the Dev session after logging out), sometimes I got the following error message instead:

```
W0919 11:42:11.676741 4131044 warnings.go:70] would violate PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "runtime" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "runtime" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "runtime" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "runtime" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")

Error connecting to the cluster. Please log in again

Updated Kubernetes config
Error connecting to the cluster. Please log in again

Error connecting to the cluster. Please log in again

Error connecting to the cluster. Please log in again

^C ✗  Dev mode interrupted by user
Cleaning resources, please wait
Failed to delete inner loop resources: deployments.apps "my-sample-go-app" is forbidden: User "system:anonymous" cannot get resource "deployments" in API group "apps" in the namespace "default"

```

Reply:
You are right. To be consistent with the other test when DevfileWatchHandler fails, I've added a test if IsForbidden